src/site/content/en/blog/quictransport/index.md

```diff
@@ -0,0 +1,320 @@
+
+### Is QuicTransport an alternative to WebRTC data channels?
+
+Yes, for client-server connections. QuicTransport shares many of the same properties as [WebRTC data channels](https://developer.mozilla.org/en-US/docs/Web/API/RTCDataChannel), although the underlying protocols are different.
+
+{% Aside %}
+WebRTC data channels support peer-to-peer communications, but QuicTransport only supports client-server connection. If you have multiple clients that need to talk directly to each other, then QuicTransport isn't a viable alternative.
+{% endAside %}
+
+Generally, running a QUIC-compatible server requires less setup and configuration than maintaining a WebRTC server, which involves understanding multiple protocols ([ICE](https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API/Connectivity#ICE_candidates), [DTLS](https://webrtc-security.github.io/#4.3.1.), and [SCTP](https://developer.mozilla.org/en-US/docs/Web/API/RTCSctpTransport)) in order to get a working transport. WebRTC entails many more moving pieces that could lead to failed client/server negotiations.
+
+The QuicTransport API was designed with the web developer use cases in mind, and should feel more like writing modern web platform code than using WebRTC's data channel interfaces. [Unlike WebRTC](https://bugs.chromium.org/p/chromium/issues/detail?id=302019), QuicTransport is supported inside of [Web Workers](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API/Using_web_workers), which allows you to perform client-server communications independent of a given HTML page. Because QuicTransport exposes a [Streams](https://streams.spec.whatwg.org/)-compliant interface, it supports optimizations around [backpressure](https://streams.spec.whatwg.org/#backpressure).
+
+However, if you already have a working WebRTC client/server setup that you're happy with, switching to QuicTransport may not offer many advantages.
+
+## Try it out
+
+The best way to experiment with QuicTransport is to use [this Python code](https://github.com/GoogleChrome/samples/blob/gh-pages/quictransport/quic_transport_server.py) to start up a compatible QUIC server locally. You can then use this page with a [basic JavaScript client](https://googlechrome.github.io/samples/quictransport/client.html) to try out client/server communications.
+
+## Using the API
+
+QuicTransport was designed on top of modern web platform primitives, like the [Streams API](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API). It relies heavily on [promises](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_promises), and works well with [<code>async</code> and <code>await</code>](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Asynchronous/Async_await).
+
+The QuicTransport [origin trial](#register-for-ot) supports three distinct types of traffic: datagrams, as well as both unidirectional and bidirectional streams.
```

Comment:
```suggestion
The QuicTransport [origin trial](#register-for-ot) supports three distinct types of traffic: datagrams, as well as both uni-directional and bi-directional streams.
```
Please do a find & replace throughout the rest of the article to make sure you're using `uni-directional` and `bi-directional` consistently

Reply:
Changed the front matter instead.